api/infra/limit.go

```diff
@@ -0,0 +1,117 @@
+//nolint:revive
```

Comment:
Where do we keep the `api_list.yaml` file?

Reply:
In generator repo